packages/plugins/src/plugins/eigenlayer/types.ts

```diff
@@ -0,0 +1,43 @@
+export type Contract = {
+  chain: string;
+  address: `0x${string}`;
```

Comment:
```suggestion
  address: Address;
```
I believe this change can be performed globally in this file (using `Address` as the type); I believe it could also simplify a lot of other castings/calls to `getAddress()`.

Let me know if it works ?

Reply:
Added `Address` globally, but I still need `getAddress()`  at some places because of the checksummed vs unchecksummed addresses comparaison, thing that `Address` isn't related too

Reply:
use `formatAddress` or `formatEvmAddress` instead of getAddress.